gimme

```diff
@@ -327,8 +327,8 @@ _compile() {
 		export CC_FOR_TARGET="${GIMME_CC_FOR_TARGET}"
 
 		local make_log="${1}/make.${GOOS}.${GOARCH}.log"
-		if [[ "${GIMME_DEBUG}" -gt "1" ]]; then
-			./make.bash 2>&1 | tee "${make_log}" 1>&2 || return 1
+		if [[ "${GIMME_DEBUG}" -ge "2" ]]; then
```

Comment:
We can keep `-gt "1"` to make tiny 3 bytes patch. 😄

Reply:
No.  Clean code is more important than a smaller patch.  People should be able to read the code and immediately see the threshold.

Reply:
About patch size – that was a joke. 😄
But I really don't get why `-ge "2"` is better than `-gt "1"`. Both are immediately visible thresholds.